tools/efficientnet2mmdet.py

```diff
@@ -0,0 +1,53 @@
+import argparse
```

Comment:
The file path should be `tools/dataset_converters/xx`

Reply:
move it to tools/model_converters/